services/movie.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Movie
+from django.db.models.query import QuerySet
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids is not None:

```

Comment:
No need to add `is not None`. What if genres_ids wiil be an empty list? Fix everywhere